docs/reference/installation.md

```diff
@@ -68,6 +69,30 @@ Then, you can install CUDA, along CuDNN.
 You can also refer to this [dockerfile](https://github.com/opendr-eu/opendr/blob/master/Dockerfile-cuda) for installation instructions.
 Note that NVIDIA 30xx GPUs may not be fully supported, due to CUDA limitations.
 
+# Installing by cloning OpenDR repository on a Nvidia-TX2
+If you are installing the toolkit on a Nvidia-TX2:
+```bash
+cd opendr
+./bin/install_tx2.sh
+```
+Note that TX2 should be flashed with Jetpack 4.6 and that this might take a while (~4-5h), while the script also makes system-wide changes and might prompt for password input.
+
+In order to use the toolkit, you should export some system variables by running `activate_tx2.sh`, (Note that this should be run before every toolkit use):
+```bash
+./bin/activate_tx2.sh
+```
+Then, you are ready to use the toolkit!
+
+The script works also on Nvidia Xavier NX and AGX. You should change line 37 in [install_tx2.sh](https://github.com/opendr-eu/opendr/blob/tx2_install/bin/install_tx2.sh) from:
+```
+-gencode arch=compute_62,code=sm_62
```

Comment:
I'm not familiar with TX2, can't we detect which board we're using and do this change programmatically? That way we can just have a generic script that handles all 3 target hardware out of the box. Otherwise I think people will just read the title, see it only mentions TX2 and assume it won't work on the others.
If it's possible I'd suggest renaming the script to something more generic as well.

Reply:
The option to choose between tx2, agx and nx is Implemented now.